src/styles/_common/_colors.scss

```diff
@@ -80,6 +80,12 @@ $palette: (
     'error': #ff6b6b,
     'info': #00adef,
     'text': #fff
+  ),
+  'footer': (
+    'background': #fafafa,
```

Comment:
I would have to agree this is a color specific mixin so the color is, itself already implied.